core/trino-main/src/main/java/io/trino/testing/assertions/Assert.java

```diff
@@ -58,6 +59,11 @@ public static void assertEventually(Runnable assertion)
     }
 
     public static void assertEventually(Duration timeout, Runnable assertion)
+    {
+        assertEventually(timeout, new Duration(50, MILLISECONDS), assertion);
+    }
+
+    public static void assertEventually(Duration timeout, Duration retryDuration, Runnable assertion)
```

Comment:
retryDuration -> retryFrequency or waitBeforeRetry

Reply:
retryFrequency is nice